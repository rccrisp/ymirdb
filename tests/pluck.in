set a 1 2 3
pluck a 0
pluck a 1
pluck a 1
pluck a 1
pluck a 1
set b -1 0
push a b
forward a
backward b
pluck a 1
pluck a 1
forward a
backward b
append a b
forward a
backward b
push a 3 2 1
pluck a 4
forward a
backward b
bye