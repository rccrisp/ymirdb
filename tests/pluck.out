> ok

> index out of range

> 1

> 2

> 3

> index out of range

> ok

> ok

> b

> a

> b

> index out of range

> nil

> nil

> ok

> b

> a

> ok

> b

> nil

> nil

> bye
